tutorials/io/plot_30_reading_fnirs_data.py

```diff
@@ -37,6 +37,21 @@
              stored in the .snirf format.
 
 
+.. _import-boxy:
+
+BOXY (.txt)
+================================
+
+BOXY recordings can be read in using :func:`mne.io.read_raw_boxy`.
+The BOXY software and Imagent devices store data in a single .txt file
+containing DC, AC, and Phase information for each source and detector
```

Comment:
This is an appropriate place to give a quite detailed description of what AC, DC, Phase are. It is still unclear to me exactly what these terms mean.